src/cascadia/TerminalApp/CommandPalette.idl

```diff
@@ -3,9 +3,21 @@
 
 import "IDirectKeyListener.idl";
 import "ShortcutActionDispatch.idl";
+import "HighlightedTextControl.idl";
 
 namespace TerminalApp
 {
+    [default_interface] runtimeclass FilteredCommand : Windows.UI.Xaml.Data.INotifyPropertyChanged
+    {
+        FilteredCommand();
+        FilteredCommand(Microsoft.Terminal.Settings.Model.Command command);
+
+        Microsoft.Terminal.Settings.Model.Command Command { get; };
+        String Filter;
```

Comment:
It is not required for the current implementation. I left this as this is "FilteredCommand", and I assumed that the interface consumers might be interested in both the Command and the Filter. I suggest to leave it, but please let me know if you think otherwise.